app/providers/jobPostProvider.ts

```diff
@@ -49,6 +88,24 @@ class JobPostProvider {
       },
     });
   }
+
+  async getAllValidJobPosts() {
```

Comment:
can you also make sure that the start date should not be before or at today. Some facility could forget to close or complete a job post so it could still be open after the start date of the job and still receive applicants if we don't filter that out

Reply:
We could keep it for now, so we this is not a factor when building the app. It would be easy to add this later on